content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Reset'
+Description: 'Used to undo commits to a Git repository.'
```

Comment:
```suggestion
Description: 'In Git, reset is used to set the HEAD pointer back to a previous commit and optionally undo staged changes and the working tree.'
```

For concept entries like this one, we can start the `Description` with a noun as opposed to a verb. (that's more for term entries like [this one](https://github.com/Codecademy/docs/blob/main/content/python/concepts/built-in-functions/terms/print/print.md))

Reply:
> this one

So like for any future entries, I should say, "In Git" or "In python" or "The " " command "..etc, if I'm not wrong?

Reply:
Yeah, for concept entries they should begin with a noun (ideally, the noun should be the concept itself). So yeah, for future concept entries you can start the `Description` with something like "In Git", "The `x` command`, etc.

Reply:
> Yeah, for concept entries they should begin with a noun (ideally, the noun should be the concept itself). So yeah, for future concept entries you can start the `Description` with something like "In Git", "The `x` command`, etc.

Thank you @Dusch4593 , is there anything more I can do?

Reply:
There are just a few more outstanding comments/suggestions that need resolves but after that we should be good to move forward. 😄

Reply:
> There are just a few more outstanding comments/suggestions that need resolves but after that we should be good to move forward. 😄

Hey! So I sort of gone through our "conversation" and crossed checked with the suggestions and what I have and it seems to be resolved. Also with regard to the pushing up of the context of the example we use, I did push it to after the options for the reset command and just before referencing commits which I hope is what you meant and I guess by that it's kinda done. Also there's a comment that I was a bit confused by and was wondering if you could clear that up for me?